xcodeproj/internal/bazel_integration_files/calculate_output_groups.py

```diff
@@ -36,9 +51,32 @@ def _calculate_output_group_target_ids(
             # Parse the build-request.json file
             build_request = json.load(f)
 
-        buildable_target_ids = []
+        # Xcode gets "suck" in the `buildFiles` or `build` command for top-level
+        # targets, so we can't reliably change commands here. Leaving the code
+        # in place in case this is fixed in the future, or we want to do
```

Comment:
Could be good to file a radar on this, and include the number in the comment.

Reply:
Could be, but I don't want to draw attention to these files, as they are not public API. I believe when we implement Build with Bazel via Proxy we will get the correct command in the API call, regardless of stale cache files.